rpc/server.go

```diff
@@ -49,26 +50,34 @@ type Server struct {
 	run      int32
 	codecs   mapset.Set
 
-	executionPool *SafePool
+	executionPool                     *SafePool
+	executionPoolThreadRequesttimeout time.Duration
 }
 
 // NewServer creates a new server instance with no registered handlers.
-func NewServer() *Server {
+func NewServer(executionPoolThreads uint64, executionPoolThreadRequesttimeout time.Duration) *Server {
 	server := &Server{
 		idgen:         randomIDGenerator(),
 		codecs:        mapset.NewSet(),
 		run:           1,
-		executionPool: NewExecutionPool(threads),
+		executionPool: NewExecutionPool(int(executionPoolThreads), executionPoolThreadRequesttimeout),
 	}
 
 	// Register the default service providing meta information about the RPC service such
 	// as the services and methods it offers.
 	rpcService := &RPCService{server}
 	server.RegisterName(MetadataApi, rpcService)
-
 	return server
 }
 
+func (s *Server) SetExecutionPoolSize(n int) {
```

Comment:
Could we also log that changes as in https://github.com/maticnetwork/bor/pull/712/files# (please ask @0xsharma for the latest changes)? It'd be great to have these logs similar.